website/content/docs/import/gcpsm.mdx

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+page_title: Google Cloud Platform Secret Manager - Secrets Import Source
+description: The Google Cloud Platform Secret Manager source imports secrets from GCP to Vault.
+---
```

Comment:
The general documentation typically uses a walkthrough style that guides the user through a full-working example from start to finish. Do you think the same pattern would be useful for Secrets Import?

Reply:
I had first copied our usual format based on other docs to base these on, but I started to notice when writing the GCP page that other source pages would be nearly identical. The only section that would be different is explaining the source block with the rest being unchanged. I figure it makes sense to have a small page for each source block, a page for the mappings, and put the destination info on the index page since we are only dealing with Vault as a destination.